lua/autorun/server/webhooker_interface.moon

```diff
@@ -0,0 +1,20 @@
+require "moonscript"
+
+webhookerUrl = (
+    ->
+        contents = file.Read "cfc/webhooker/url.txt", "DATA"
+        string.gsub contents, "%s", ""
```

Comment:
Does this remove whitespace?

Reply:
Yes, especially that pesky `\n` at the end of read files